Scripts/extract_release_notes.rb

```diff
@@ -56,31 +58,30 @@ def replace_pr_number_with_markdown_link(string)
 # sequence. This accounts for the edge case in which more new lines make it
 # into the release notes file than expected.
 index = 0
-until lines[index].start_with? '-----'
-  index += 1
-end
+index += 1 until lines[index].start_with? '-----'
 
-lines[(index+1)...].each do |line|
+lines[(index + 1)...].each do |line|
   break if line.strip == ''
+
   release_lines.push line
 end
 
-formatted_lines = release_lines.
-    map { |l| l.gsub(/-   /, '- ') }
+formatted_lines = release_lines
+                  .map { |l| l.gsub('-   ', '- ') }
 
 case mode
 when :strip_pr_links
-  formatted_lines = formatted_lines.
-    map { |l| l.gsub(/ \#\d*$/, '') }
+  formatted_lines = formatted_lines
+                    .map { |l| l.gsub(/ \#\d*$/, '') }
 when :keep_pr_links
   formatted_lines = formatted_lines.
-    # The PR "links" are not actually links, but PR "ids". On GitHub, they'll
-    # be automatically parsed into links to the corresponding PR, but outside
-    # GitHub, such as in our internal posts or on App Center, they won't.
-    #
-    # It's probably best to update the convention in writing the release notes
-    # but in the meantime let's compensate with more automation.
-    map { |l| replace_pr_number_with_markdown_link(l) }
+                    # The PR "links" are not actually links, but PR "ids". On GitHub, they'll
+                    # be automatically parsed into links to the corresponding PR, but outside
+                    # GitHub, such as in our internal posts or on App Center, they won't.
+                    #
+                    # It's probably best to update the convention in writing the release notes
+                    # but in the meantime let's compensate with more automation.
+                    map { |l| replace_pr_number_with_markdown_link(l) }
```

Comment:
💄  This makes some odd indentation and formatting… which is also not consistent with the formatting used in the previous `when` case of having the `.` in `.map` be on the next line.

I'm guessing the auto-correct fixed it that way because of the comment above `map`; but it would probably be nicer and more consistent to just move the comment one line up and then use the same formatting as the other `when` case?

```suggestion
  # The PR "links" are not actually links, but PR "ids". On GitHub, they'll
  # be automatically parsed into links to the corresponding PR, but outside
  # GitHub, such as in our internal posts or on App Center, they won't.
  #
  # It's probably best to update the convention in writing the release notes
  # but in the meantime let's compensate with more automation.
  formatted_lines = formatted_lines
    .map { |l| replace_pr_number_with_markdown_link(l) }
```

Reply:
Fixed in https://github.com/Automattic/simplenote-ios/commit/026291a474a6688590b72f95387b8ad015af6ec8